bin/tx-prover/README.md

```diff
@@ -114,6 +114,16 @@ The proxy service uses this health check to determine if a worker is available t
 
 Both the worker and the proxy will use the `info` log level by default, but it can be changed by setting the `RUST_LOG` environment variable.
 
+## Traces
```

Comment:
I would maybe combine this and logging into one section (or make this a sub-section of logging?).

Also, I would add more details here. For example, it is not clear where tracing/logging info is written to. Is it `stdout`? Is it some logging file? Somewhere else? It is also not clear whether there is a way to not use Jaeger (or maybe use something else) to view the logs.

Basically, a bit more context about how tracing/logging works would be helpful.